src/interpreter/ByteCodeGenerator.cpp

```diff
@@ -183,7 +185,7 @@ ByteCodeBlock* ByteCodeGenerator::generateByteCode(Context* context, Interpreted
     ByteCodeGenerateContext ctx(codeBlock, block, codeBlock->isGlobalScope(), codeBlock->isEvalCode(), inWithFromRuntime || codeBlock->inWith(), nData);
 
 #ifdef ESCARGOT_DEBUGGER
-    ByteCodeBreakpointContext breakpointContext;
+    ByteCodeBreakpointContext breakpointContext(context->debugger() && context->debugger()->parsingEnabled());
```

Comment:
One more question,
Why don't you check `context->debugger()->enabled()` here?

Reply:
No need to. This flag is independent from `enabled()`. But it can be if you prefer it.

Reply:
Ok I see.